assets/css/src/amp-validation-error-taxonomy.css

```diff
@@ -362,3 +364,138 @@ body.taxonomy-amp_validation_error .wp-list-table .new th.check-column input {
 	font-weight: bold;
 	text-align: right;
 }
+
+/* Stylesheets */
+.amp-stylesheet-summary {
+	margin-bottom: 1em;
+}
+
+.amp-stylesheet-summary th {
+	text-align: right;
+	padding-right: 1ex;
+	font-weight: normal;
+}
+
+.amp-stylesheet-summary td {
+	font-weight: bold;
+	text-align: right;
+}
+
+.stylesheet-details {
+	display: none;
+}
+
+.stylesheet.expanded + .stylesheet-details {
+	display: table-row;
+}
+
+.stylesheet-details dl.detailed {
+	margin-bottom: 0;
+	padding-bottom: 0;
+}
+
+.shaken-stylesheet {
+	display: block;
+	white-space: pre-wrap;
+	overflow-x: scroll;
```

Comment:
Is this needed? In my (brief) testing I was not able to find a stylesheet that overflowed the edges, and instead rendered a disabled horizontal scroll bar and a vertical scroll bar:

![image](https://user-images.githubusercontent.com/16200219/72782102-d777f280-3c1a-11ea-9ee1-eb9ad3a25f84.png)

Reply:
I'm not seeing that horizontal scrollbar in either Chrome or Firefox. 

You're right, though. This should not be necessary because of `white-space: pre-wrap;` and also because each property is now on a separate line after e0055b5.

Removed in da85b8f49905f3dcf0ebc30c6716a9f35c97b556.